### PR TITLE
Update build app page

### PR DESCRIPTION
**Pull request by EstherPhebe**
addresses odk-x/tool-suite-X#207

#### What is included in this PR?
1. Remove reference to grunt file for tables as it is currently not working
2. Add Skip logic table as List and Detail view example

#### What new issues will need to be opened because of this PR?
none
#### What is left to be done in the addressed issue?
none
#### What problems did you encounter?
none

### CODE REVIEW

File: odkx-src/build-app.rst [hunk 1]
```diff
@@ -66,19 +66,21 @@ When creating a new form, the appropriate directory structure must be created. O
...
+  1. Navigate to :file:`app/config/tables/` and create a folder with the tableId, where tableId is the name of your new form and table. For example, to create a census form; the folder would be named census.
 
-  $ grunt addtable:census
+  2. In the census folder, create the following new folders:
+   
+    - :th:`forms`
+    - :th:`html`
+    - :th:`js`
+    - :th:`instances`
+    - :th:`collect-forms`
```
Comment (elmps2018) [inline odkx-src/build-app.rst:82]: I am not sure all these folders are needed... forms definitely. Are the others needed for tables? I mostly use survey so not sure.

**Review by elmps2018 — COMMENTED** [review]
This looks great (so far as I could tell! Someone who knows tables code better should also check!)

File: odkx-src/build-app.rst [hunk 2]
```diff
@@ -50,9 +50,9 @@ Before getting started, be sure you have familiarized yourself with the ODK-X pl
 Cleaning App Designer
 -----------------------------------
 
-Your freshly installed copy of Application Designer comes with lots of example forms, tables, and configuration. This is useful for learning the tools and as references when building our application, but the files should be cleaned before building your own application.
+Your freshly installed copy of Application Designer comes with lots of example forms, tables, and configuration. This is useful for learning the tools and as references when building our application, the files can be found in :file:`app/config/tables` directory. 
 
-Enter your Application Designer directory, navigate to :file:`app/config/` and delete everything inside the directory.
+After building your own application, you may choose to delete all the examples forms and configurations before pushing your files to your device; as the files can be very large and take up a lot of space on the device.
```
[the remark this answers is not in the document]
Comment (wbrunette) [reply odkx-src/build-app.rst:56]: I am referring to deleting all the example forms. It was automated to avoid confusion (you should test it).

**Comment by wbrunette** [comment]
@EstherPhebe I added comments to Caroline's review

File: odkx-src/build-app.rst [hunk 3]
```diff
@@ -55,6 +55,12 @@ Your freshly installed copy of Application Designer comes with lots of example f
 
 After building your own application, you may choose to delete all the examples forms and configurations before pushing your files to your device. The files can be very large and take up a lot of space on the device.
 
+Open the terminal and type *grunt empty*, this removes all the files and creates an empty app-designer folder to work with.
```
Comment (elmps2018) [inline odkx-src/build-app.rst:58]: Suggest adding "To delete all the example forms and configurations, open"

File: odkx-src/build-app.rst [hunk 4]
```diff
@@ -784,24 +794,27 @@ If you pull the CSV files, they will be under the :file:`output/csv/` directory.
...
+:guilabel:`grunt adbpull-csv` : Pull any exported csv files from the device.
 
-:guilabel:`grunt setup` : Launches the login and sync screen on the connected device.
+:guilabel:`grunt adbpush` : Perform all the Android Debug Bridge push tasks.
 
-:guilabel:`grunt kill all` : Force stops survey, tables and services on the connected device.
+:guilabel:`grunt addtable:tableid` : Will create the required directory structure for an individual table, including the forms directory.
 
-:guilabel:`grunt uninstall` : Uninstall ODK-X tools from the connected device.
+:guilabel:`grunt clean` : Wipes the device of all ODK-X data
```
Comment (elmps2018) [inline odkx-src/build-app.rst:807]: for consistency add period.

**Review by elmps2018 — CHANGES_REQUESTED** [review]
One tiny copy edit otherwise looks good to me!

**Comment by elmps2018** [comment]
@EstherPhebe there is now a conflict too...

**Comment by EstherPhebe** [comment]
> @EstherPhebe there is now a conflict too...

I'll fix that.

**Comment by EstherPhebe** [comment]
I'm sorry about the weird commit history. I did a git rebase to fix the conflict

**Review by elmps2018 — APPROVED** [review]

**Comment by elmps2018** [comment]
@EstherPhebe this looks fabulous to me!

**Pull request merged**